types/src/system/mint/error.rs

```diff
@@ -25,10 +25,10 @@ pub enum Error {
     /// Destination purse not found.
     #[cfg_attr(feature = "std", error("Destination not found"))]
     DestNotFound = 2,
-    /// See [`PurseError::InvalidURef`].
+    /// See `PurseError::InvalidURef`.
```

Comment:
These are unhelpful as `PurseError` is private.  Let's move the comments currently on the `PurseError` variants to here, and have the `PurseError` doc comments refer the reader to here to avoid duplicating them.